### ISSUE TITLE
Fix active link for home page

### ISSUE THREAD

**Issue by jmeas** [orbitjs/orbitjs.com#18]
The home page link is active on every subpage because of the method I'm using to set it as active.

**Comment by jmeas** [comment]
Notes for myself:

`page.url` === `/guides/index.html`
`item.url` === `/guides/`

**Comment by jmeas** [comment]
The solution is:

```js
  <ul>
    {% assign url = page.url|remove:'index.html' %}
    {% for item in site.data.navigation %}
      <li {% if url == item.url %}class="active"{% endif %}>
        <a href='{{ item.url }}'>{{ item.title }} {{ page.url }}, {{ item.url }}</a>
      </li>
    {% endfor %}
  </ul>
```

**Comment by dgeb** [comment]
That looks like it should work. :+1: 

@jmeas are there any alternatives to Liquid templates? I haven't used them in years but remember them as having a lot of constraints. (Please disregard if you're already far down the path and Liquid is working well for you).

**Comment by jmeas** [comment]
> @jmeas are there any alternatives to Liquid templates?

Hm, not that I'm aware of. But I haven't looked too extensively. My understanding of Jekyll is that it's sort of an all-or-nothing thing. I'll investigate in the coming days, though, if just for my own edification.

> I haven't used them in years but remember them as having a lot of constraints. (Please disregard if you're already far down the path and Liquid is working well for you).

I can't say that it's my favorite templating language in the world, but it's also not my least favorite, or anything like that. If it came down to getting rid of Jekyll entirely to replace the Liquid templates, I don't think that that would be a worth it.

**Comment by dgeb** [comment]
Thanks @jmeas - good to know. I've primarily used Middleman in the past and am less familiar with Jekyll and its options.

**Comment by jmeas** [comment]
Fixed in 012ee50294edb1540911a9cff3b3150f9bd0bc95!
